docs/requirements.txt

```diff
@@ -1,4 +1,3 @@
 sphinx
 sphinx-argparse
 sphinx_rtd_theme
--e .
```

Comment:
Having this included was giving me:
```
ERROR: file:///home/bfauble/BryansGreatWorkspace/synapsePythonClient/docs (from -r requirements.txt (line 4)) does not appear to be a Python project: neither 'setup.py' nor 'pyproject.toml' found.
```

Also - the pre-commit wanted to move this to line 1 - which is what prompted me to look at this.

Reply:
I think this was included only for a test read the docs integration.

Let's revisit the doc deployment strategy later on. It's an inconvenience that we have to build it ourselves when deploying - and our build doesn't allow for deployment of docs of past versions on our site.